test/unit/sthTestUtils.js

```diff
@@ -912,21 +912,22 @@ function aggregatedDataAvailableSinceDateTest(ngsiVersion, params, done) {
             break;
     }
 
-    let value;
```

Comment:
Hi @fgalan,

I have added the new test cases [here](https://github.com/MadhuNEC/fiware-sth-comet/blob/issue432_aggrMethod_partial/test/unit/sth_test.js#L327).

Previously, only one aggrMethod was there so switch case was working fine. But in case of multiple aggregation method we need to update the code for multiple values separately. The above code change is for util class to accommodate the multiple aggregation methods.